awx/ui_next/src/api/models/Organizations.js

```diff
@@ -8,11 +8,11 @@ class Organizations extends InstanceGroupsMixin(NotificationsMixin(Base)) {
     this.baseUrl = '/api/v2/organizations/';
   }
 
-  readAccessList(id, params = {}) {
+  readAccessList(id, params) {
     return this.http.get(`${this.baseUrl}${id}/access_list/`, { params });
```

Comment:
This will create an object like:

```
{
  params: { ... }
}
```

Should this be spread `{ ...params }` instead?

Reply:
this is what seems to work, I think the api or axios expects it on this key.  things on this key run through the paramsSerializer function set up in base.